Source/Shaders/Builtin/Structs/materialInput.glsl

```diff
@@ -25,4 +25,5 @@ struct czm_materialInput
     float height;
     float slope;
     float aspect;
+    vec4 layerColor[czm_layerColorNumber];
```

Comment:
Just confirming - to check that a layer exists would the material have code like `materialInput.layerColor[index] != vec4(0.0)`?

`layerColor` should be added to the documentation above.

Reply:
This can be a way to test it, indeed. If ever the number of layers passed to the material could be set by the user, it would be more obvious which layer colors are made available to the material.